<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QDialog" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1213</width>
    <height>811</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="UserLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Username: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="UserLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>221</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="UsernameCheck">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>60</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff2600;&quot;&gt;Check if username exits (redfont)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PassLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>221</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="PassLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="RetypeLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Re-type Password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="RetypeLineEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>231</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="CheckPassLabel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>160</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff2600;&quot;&gt;Check if Password match (redfont)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="BirthdayBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>401</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Date of Birth: </string>
   </property>
   <widget class="QLabel" name="MonthLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Month:</string>
    </property>
   </widget>
   <widget class="QLabel" name="DayLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Day:</string>
    </property>
   </widget>
   <widget class="QLabel" name="YearLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Year:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="MonthComboBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>82</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>January</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>February</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>March</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>April</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>May</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>June</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>July</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>August</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>September</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>October</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>November</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>December</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="DayComboBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>82</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>13</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>17</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>18</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>21</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>22</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>23</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>24</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>25</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>26</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>27</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>28</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>29</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>31</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="YearComboBox">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>82</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2024</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2023</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2022</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2021</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2020</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2019</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2018</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2017</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2016</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2015</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2014</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2013</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2012</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2011</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2010</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2009</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2008</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2007</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2006</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2005</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2004</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2003</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2002</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2001</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1999</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1998</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1997</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QLabel" name="CheckAgeLabel">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>310</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff2600;&quot;&gt;Check age (redfont)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="GenderGroupBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>370</y>
     <width>191</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Gender:</string>
   </property>
   <widget class="QRadioButton" name="FemaleRadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>112</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Female</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="MaleRadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>112</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Male</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="AccTypeGroupBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>369</y>
     <width>191</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Account Type:</string>
   </property>
   <widget class="QRadioButton" name="UserRadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>112</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>User</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="AdminRadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>112</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Admin</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="FavGenreGroupBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>489</y>
     <width>331</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Favorite Genre(s)</string>
   </property>
   <widget class="QCheckBox" name="ActionCheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>93</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Action</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="DramaCheckBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>93</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Drama</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ComedyCheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>93</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Comedy</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="RomanceCheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>93</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Romance</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="HorroCheckBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>93</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Horror</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="OtherCheckBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>93</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Other</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="RegisterButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>670</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QLabel" name="CheckRegisterFields">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>680</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Check if all fields are full</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
